src/Orders/OrderScreen.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable no-plusplus */
+import React, { useEffect, useState } from 'react';
+import {
+  View, StyleSheet, ScrollView, Alert,
+} from 'react-native';
+import { Title } from 'react-native-paper';
+import PropTypes from 'prop-types';
+import Config from 'react-native-config';
+import OrderCard from './OrdersCard';
+
+const Airtable = require('airtable');
+
+const airtableConfig = {
+  apiKey: Config.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: Config.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey })
+  .base(airtableConfig.baseKey);
+
+const CONST_USER = 'helen@gmail.com';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+  },
+  centeredContainer: {
+    marginTop: '10%',
+  },
+  titleText: {
+    marginBottom: 20,
+    fontSize: 30,
+  },
+  orderCardsContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    flexWrap: 'wrap',
+    alignContent: 'center',
+    justifyContent: 'space-around',
+  },
+});
+
+export default function OrderScreen({ navigation }) {
+  const [orderMap, setOrderMap] = useState(new Map());
+  const [itemsList, setItemsList] = useState(new Map());
+  const [cardList, setCardList] = useState([]);
+  const [flag, setFlag] = useState(false);
+
+  const getOrders = () => {
+    const condenseOrders = new Map();
+    let itemsListVar = new Map();
+
+    base('Orders').select({ filterByFormula: `({user_id}='${CONST_USER}')` }).eachPage((records, fetchNextPage) => {
+      records.forEach((record) => {
+        const order = record;
+        order.fields.orderId = order.id;
+        if (!('delivery date (temp)' in record.fields)) {
+          order.fields['delivery date (temp)'] = 'September 5, 2022';
+        }
```

Comment:
This is providing a default date if an entry in the order table doesn't have a value. We will never encounter this situation since, the order table is auto populated by our checkout code, which always provides a date.

I modified the code to instead return, and skip that record if a date isn't provided. This'll just ensure it doesn't display incorrect data in the past orders screen. Does that sound ok?

Reply:
yup sounds good :D